<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wordLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>600</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 40pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>hello</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="meaningLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>你好</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="numDisplayLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>82</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="formerPushButton">
       <property name="text">
        <string>上一个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPushButton">
       <property name="text">
        <string>下一个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="meaningPushButton">
       <property name="text">
        <string>显示意思</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="collectPushButton">
       <property name="text">
        <string>收藏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="randomPushButton">
       <property name="text">
        <string>随机排序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renewPushButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="modeDisplayLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="addNewWordAction"/>
    <addaction name="skipToAction"/>
    <addaction name="updateAction"/>
    <addaction name="editAction"/>
    <addaction name="switchModeAction"/>
    <addaction name="clearAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addNewWordAction">
   <property name="text">
    <string>add new word</string>
   </property>
  </action>
  <action name="skipToAction">
   <property name="text">
    <string>skip to</string>
   </property>
  </action>
  <action name="updateAction">
   <property name="text">
    <string>update word list</string>
   </property>
  </action>
  <action name="editAction">
   <property name="text">
    <string>edit</string>
   </property>
  </action>
  <action name="switchModeAction">
   <property name="text">
    <string>switch mode</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>clear the &quot;review&quot;</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
